svm/src/account_loader.rs

```diff
@@ -48,10 +51,56 @@ pub struct LoadedTransaction {
     pub rent_debits: RentDebits,
 }
 
-pub type TransactionLoadResult = (Result<LoadedTransaction>, Option<NonceFull>);
-pub type TransactionCheckResult = (transaction::Result<()>, Option<NoncePartial>, Option<u64>);
+pub fn validate_fee_payer(
```

Comment:
This function and `load_accounts` below are also public. Should we write a doc comment for them?

Reply:
good point. I was postponing adding doc comments for non-trivial functions to have a better understanding of their logic. I'll add brief comments with an intent to extend the doc comments at a later time.

Reply:
Added the corresponding comments. I'm sorry, I augmented the previous commit, instead of making a new commit. Also rebased to the latest master.